rowan/src/Rowan-Services-Tests/JadeServerTest.class.st

```diff
@@ -98,7 +100,7 @@ JadeServerTest >> test_updateFromSton [
 
 	| service stonString resultString services |
 
-	self assert: ((UserGlobals at: #JadeServer) canUnderstand: #updateFromSton:).
+	self assert: JadeServer canUnderstand: #updateFromSton:.
```

Comment:
@dalehenrich Only one minor problem after getting your code. ``assert:canUnderstand:`` is ironically not understood. I have added parens and will check it in. fyi

Reply:
... since I didn't have a list of the known to pass RowanService tests I couldn't tell typo failures form the other kind ... so I didn't look at the exact cause of the test failure ... there could be more typos on the way:)